UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Changeling/ChangelingAbility/Implementation/Sting.cs

```diff
@@ -35,11 +35,11 @@ protected PlayerScript GetPlayerOnClick(ChangelingMain changeling, Vector3 click
 				target = integrity;
 				break;
 			}
-			if (target == null || target.Mind == null)
+			if (target == null || target.Mind == null || target.playerHealth.IsDead)
```

Comment:
so what will happen now when the target is dead? Will it just do nothing? Perhaps it should communicate the changeling they can't absorb dead people.

Reply:
Good idea

Reply:
Yep, it just do nothing currently